lib/drivers/carelinkDriver.js

```diff
@@ -55,7 +57,7 @@ function initializeProcessors(opts) {
   // re-ordering.
   return [
     require('../carelink/settings.js')(opts),
-    require('../carelink/smbg.js')(opts),
+    require('../carelink/smbg.js')(_.assign(opts, csvFormat.pumpTypesToRead.BGCapturedOnPump)),
```

Comment:
Would handle this more elegantly in the final version I hope - this is just my hack for doing just one of the types this way.